config/docker/notify.sh

```diff
@@ -22,7 +22,7 @@ if [ "$create_config_only" = "true" ]; then
   exit 0
 fi
 
-HASH=$(md5sum $(readlink -f $file))
+HASH=""
```

Comment:
@raffaelespazzoli This caused a bug where configs don't load on startup so it attempts to run with the basic defaults. i.e.

```Wed Nov  9 16:24:44 2022: Opening file '/etc/keepalived.d/keepalived.conf'.
Wed Nov  9 16:24:44 2022: (/etc/keepalived.d/keepalived.conf: Line 14) number '0' outside range [1, 255]
Wed Nov  9 16:24:44 2022: (/etc/keepalived.d/keepalived.conf: Line 14) (ingress-nginx/ingress-nginx-controller): VRID '0' not valid - must be between 1 & 255```